.github/workflows/build_wheels.yml

```diff
@@ -27,8 +27,9 @@ on:
     tags: 'v*'
   pull_request:
     branches: ['master', 'release-*']
-    tags: 'v*'
-    paths: ['sdks/python/**', 'model/**', 'release/**']
+    # TODO(pabloem): DO NOT MERGE. Commenting out to test.
```

Comment:
leftover?

Reply:
Yeah, I left that to make sure the workflow would run. Since it's run, now I'll remove that commit.